app/models/repository.rb

```diff
@@ -7,6 +7,7 @@ class Repository < ApplicationRecord
 
   scope :only_installer_updates, -> { unscope(where: :installer_updates).where(installer_updates: true) }
```

Comment:
@ikapelyukhin any idea why we `unscope` here? is it really necessary?

Reply:
Yeah, unscoping and then scoping looks weird.
My best guess is that it is a weird artifact after porting the code from glue (there it has a default scope, which has to be cleared out by default).
Let's try removing it :-)